notices/notice-nonce.json

```diff
@@ -1 +1 @@
-1
+2
```

Comment:
If we delete notices ever, we want the ids to keep incrementing, so this was my solution to make sure we didn't re-use ids.